src/frontend/js/components/AddressesManagement/ValidationErrors.ts

```diff
@@ -0,0 +1,37 @@
+import { defineMessages } from 'react-intl';
+
+export enum ErrorKeys {
```

Comment:
I needed to extract these values into a new `ValidationErrors.ts` file because `getLocalizedErrorMessage` was moved into `validationSchema.ts` and for testing purpose `AddressForm.spec.tsx` needs to mock the `errorMessages` below. So `getLocalizedErrorMessage` AND `errorMessages`needs to be in distinct files to make the mock work.

Reply:
You are able to mock only some exports of a module but anyway I think it makes sense to move those stuff in a dedicated module.